### PR TITLE
docs: made sure that a pitfall of `make_sdf` is documented

### PR DESCRIPTION
**Pull request by CommanderStorm**
I had this problem during https://github.com/maplibre/martin/pull/1492
Adding a bit of documentation here might safe future devs a bit of time ^^

### CODE REVIEW

File: src/sprite/mod.rs [hunk 1]
```diff
@@ -352,6 +352,10 @@ impl SpritesheetBuilder {
         self
     }
 
+    /// Adds the metadata that all images are sdf sprites.
+    /// 
+    /// You have to ensure that the Sprites are created as a sdf file beforehand.
+    /// See [`Sprite::new_sdf`] for further context.
```
Comment (flother) [inline src/sprite/mod.rs:358]:
```suggestion
    /// Add metadata to indicate that all images are SDF sprites.
    ///
    /// You have to ensure that the sprites are created as an SDF file beforehand. See
    /// [`Sprite::new_sdf`] for further context.
```

**Review by flother — CHANGES_REQUESTED** [review]
This is very useful, thanks for taking the time to add it. Do you mind if I ask for a few tweaks? Use the imperative mood, capitalise acronyms, and use a line-length of 100 chars, that sort of thing

File: src/sprite/mod.rs (hunk 1 above)
Comment (CommanderStorm) [reply src/sprite/mod.rs]: > Do you mind if I ask for a few tweaks

This is your repository xD

Sorry about the line lenghts, I just did it quick-and-dirty in githubs editor

**Review by flother — APPROVED** [review]
Thanks!

**Pull request merged**